drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableModellerPresenter.java

```diff
@@ -405,4 +407,9 @@ public void updateLinks() {
         }
         getView().getGridLayerView().refreshGridWidgetConnectors();
     }
+
+    @Override
+    public void addVerificationPanel(AnalysisReportScreen analysisReportScreen) {
```

Comment:
Just for consistency, could we use `final AnalysisReportScreen analysisReportScreen`